src/libs/getDefaultStateForField.js

```diff
@@ -0,0 +1,16 @@
+import lodashGet from 'lodash/get';
+
+/**
+ * Add / to the end of any URL if not present
+ * @param {Object} props
+ * @param {String} fieldName
+ * @param {*} defaultValue
+ *
+ * @returns {*}
+ */
+function getDefaultStateForField(props, fieldName, defaultValue = '') {
+    return lodashGet(props, ['reimbursementAccountDraft', fieldName])
+        || lodashGet(props, ['achData', fieldName], defaultValue);
```

Comment:
Can you rename this file to be more specific or move it to something like ValidateBankAccountUtils.js? `getDefaultStateForField` doesn't imply that the fieldNames are nested under achData/reimbursementAccount

Reply:
Renamed!